webpack.config.js

```diff
@@ -50,7 +61,7 @@ module.exports = () => {
               {
                 loader: 'css-loader',
                 options: {
-                  modules: true,
+                  modules: false,
```

Comment:
Ok, so my understanding is that this is https://github.com/css-modules/css-modules . I didn't even know we could have scopes in CSS :P

Reply:
Well, actually, CSS modules are webpack extension. :P